src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -120,11 +120,13 @@ bipyramid(P::Polyhedron{T}, z::Number, z_prime::FieldElem) where {T<:scalar_type
 
 @doc raw"""
     normal_cone(P::Polyhedron, i::Int64)
```

Comment:
```suggestion
    normal_cone(P::Polyhedron, i::Int64; outer::Bool = false)
```
Adjust to added `outer` argument (see further down).